### ISSUE TITLE
no-deprecate-event-creation incorrectly flags e.force:showToast

### ISSUE THREAD

**Issue by nwcm** [forcedotcom/eslint-plugin-aura#39]
![image](https://user-images.githubusercontent.com/111259588/234453633-03538cf2-99eb-47e5-9b67-ac8e123b5ce7.png)

As per Salesforce documentation, and functionality the below is valid
``` apex
const toastEvent = $A.get("e.force:showToast");
```
https://developer.salesforce.com/docs/component-library/bundle/force:showToast/documentation
